_posts/wiki/2020-03-23-robotic-skin.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Robotic Skin
+description: Research Page
```

Comment:
better subtitle needed

Reply:
How about **Outline of the three main artificial skin contributions: skin unit design, automatic kinematic calibration, and collision avoidance**